packages/@ourworldindata/grapher/src/stackedCharts/StackedDiscreteBarChart.tsx

```diff
@@ -483,6 +487,8 @@ export class StackedDiscreteBarChart
             timing: { duration: 350, ease: easeQuadOut },
         })
 
+        const yOffset = barSpacing / 2
```

Comment:
Ah, for `StackedDiscreteBarChart` you can use the `topYOffset`, which is already defined, and get rid of the `transform` again.